### PR TITLE
[13.0][ADD] product_pack_category_restriction

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Module to restrict pack products to specific product categories.

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
Hi @ernestotejeda and @pedrobaeza 

Could I get your feedback on this new module? :)

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
I think a better name would be `product_pack_category_restriction`

**Comment by JordiMForgeFlow** [comment]
I agree @pedrobaeza, name has been now changed. Thank you for the feedback :)

File: .copier-answers.yml [hunk 1]
```diff
@@ -14,7 +14,8 @@ include_wkhtmltopdf: false
 odoo_version: 13.0
 org_name: Odoo Community Association (OCA)
 org_slug: OCA
-rebel_module_groups: []
+rebel_module_groups:
+- product_pack_category_restriction
```
Comment (pedrobaeza) [inline .copier-answers.yml]: Don't put this. The defaults should be compatible (see later).

File: product_pack_category_restriction/models/product_category.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    pack_ok = fields.Boolean(string="Is Pack Category?")
```
Comment (pedrobaeza) [inline product_pack_category_restriction/models/product_category.py]: The default should be compatible.

```suggestion
    pack_ok = fields.Boolean(string="Is Pack Category?", default=True)
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by JordiMForgeFlow** [comment]
Makes sense @pedrobaeza , thanks!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**